crates/graphql-composition/src/ingest_subgraph/schema_definitions/link.rs

```diff
@@ -154,9 +150,20 @@ fn ingest_grafbase_extension_from_link(
         let Some(name) = r#as.or_else(|| {
             let mut segments = link_url.url.path_segments()?;
 
-            segments.next_back()?;
+            let name = segments.next_back()?;
```

Comment:
Just in case, I'm changing this to use the same name as the link

Reply:
Yeah, I can remove that change, doesn't matter.